server/server.go

```diff
@@ -83,6 +83,8 @@ func (s Server) Serve(port int) error {
 	v1.HandleFunc("/discover/tv", s.SearchTV()).Methods(http.MethodGet)
 
 	v1.HandleFunc("/indexers", s.ListIndexers()).Methods(http.MethodGet)
```

Comment:
This ListIndexers is going to return the in memory ones still right? Not that we have to fix it right away

Reply:
yeah, wasn't sure if we wanted to cut over from using prowlarr or not.. but could make the swap to store prowlarr in the db and then use that for the indexer lookups too